web/apps/labelstudio/src/config/ApiConfig.example.js

```diff
@@ -0,0 +1,139 @@
+/* 
+to mock a request, you can define the endpoint with a schema that looks like this:
```

Comment:
it is not clear from description whether it replaces the default ApiConfig or only partially replaces the endpoints provided. Can you elaborate on details how it works and it is supposed to be used?